packages/autorest.go/test/acr/azacr/fake/zz_containerregistry_server.go

```diff
@@ -213,15 +213,15 @@ func (c *ContainerRegistryServerTransport) dispatchCreateManifest(req *http.Requ
 	if err != nil {
 		return nil, err
 	}
+	if val := server.GetResponse(respr).ContentLength; val != nil {
```

Comment:
In the constructor for `StructDef` we sort the properties.  This sorting was not preserved in the previous implementation.